student.rb

```diff
@@ -1,4 +1,4 @@
-require './classroom'
+require_relative './classroom'
 class Student < Person
   attattr_accessor :@classroom
 
```

Comment:
- [ ]  This is actually a wrong **getter** syntax, it returns this error  `undefined method `attattr_accessor' for Student:Class (NoMethodError)`. The correct one is `attr_accessor :classroom `. Please fix it